src/cow-react/common/utils/isOrderCancellable.test.ts

```diff
@@ -0,0 +1,58 @@
+import { Order, OrderStatus } from 'state/orders/actions'
+import { NATIVE_CURRENCY_BUY_TOKEN } from 'custom/constants'
+import { isOrderCancellable } from './isOrderCancellable'
+import { COW } from 'constants/tokens'
+
+describe('isOrderCancellable', () => {
+  it('When order cancellation in progress, the it cannot be cancelled', () => {
+    const order = {
+      inputToken: COW[1],
+      status: OrderStatus.PENDING,
+      isCancelling: true, // <-----
+      cancellationHash: undefined,
+    } as Order
+
+    expect(isOrderCancellable(order)).toBe(false)
+    expect(isOrderCancellable({ ...order, isCancelling: false })).toBe(true)
```

Comment:
nit: not sure why you test the negative too inside this test. In principle, that should be other test

Reply:
Corresponding to the best practices - yes, it should. But the current version is two times shorter and the readability is not worse, isn't?